webpack.config.js

```diff
@@ -4,16 +4,25 @@ const CopyPlugin = require('copy-webpack-plugin');
 
 module.exports = {
   mode: 'development',
-  entry: {
-    main: './src/index.js',
-  },
+  entry: './src/index.js',
```

Comment:
removed chunking and splitting for now. The project is small enough that I dont think we need it

Reply:
agreed